assignments/array-methods.js

```diff
@@ -56,21 +56,34 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
+runners.forEach(runner => {fullName.push(`${runner.first_name} ${runner.last_name}`)});
```

Comment:
remember one of the powerful things arrow functions let you do is leave out the curly braces in your function blocks!